cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/elements/SynchronousMachineEq.java

```diff
@@ -47,4 +49,9 @@ public static void write(String id, String generatorName, String equipmentContai
 
     private SynchronousMachineEq() {
     }
+
+    private static double defaultRatedS(double minP, double maxP, double minQ, double maxQ) {
```

Comment:
We have asked some help at ENTSOE side. We have tried and it works. See below the code we have to do:
`
            List<Double> values = new ArrayList<>();
            values.add(Math.abs(g.getMinP()));
            values.add(Math.abs(g.getMaxP()));
            ReactiveLimits limits = g.getReactiveLimits();
            if (limits.getKind() == ReactiveLimitsKind.MIN_MAX) {
                values.add(Math.abs(g.getReactiveLimits(MinMaxReactiveLimits.class).getMinQ()));
                values.add(Math.abs(g.getReactiveLimits(MinMaxReactiveLimits.class).getMaxQ()));
            } else { // reactive capability curve
                ReactiveCapabilityCurve curve = g.getReactiveLimits(ReactiveCapabilityCurve.class);
                for (ReactiveCapabilityCurve.Point p : curve.getPoints()) {
                    values.add(Math.abs(p.getP()));
                    values.add(Math.abs(p.getMinQ()));
                    values.add(Math.abs(p.getMaxQ()));
                }
            }
            values.sort(Double::compareTo);
            double max = values.get(values.size() - 1);
            g.setRatedS(max);
`

Reply:
Yes, we prepared the changes taking into account the minP, maxP, minQ, maxQ as described by ENTSO-E in the [Quality of CGMES datasets and calculations v3.3 ](https://eepublicdownloads.azureedge.net/clean-documents/digital/QualityOfCGMESdatasetsAndCalculations_v3_3.pdf). See the relevant doc for rule `SMRatedSunrealistic`:
<img width="723" alt="Screenshot 2023-05-29 at 10 47 03" src="https://github.com/powsybl/powsybl-core/assets/3374819/d40df3a1-209e-46eb-a158-91c880700502">
<img width="737" alt="Screenshot 2023-05-29 at 10 46 36" src="https://github.com/powsybl/powsybl-core/assets/3374819/9ddd1147-7bb4-450b-9004-d6ef4939eed3">

Reply:
Actually, I added two conditions in my code: 
````
values.add(Math.sqrt(p.getP() * p.getP() + p.getMinQ() * p.getMinQ()));
values.add(Math.sqrt(p.getP()` * p.getP() + p.getMaxQ() * p.getMaxQ()));
````
This is to follow the rule `RCCXValues4` from QoCDC.